.changeset/config.json

```diff
@@ -6,5 +6,8 @@
   ],
   "access": "public",
   "baseBranch": "main",
-  "updateInternalDependencies": "patch"
+  "updateInternalDependencies": "patch",
+  "___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH": {
+    "onlyUpdatePeerDependentsWhenOutOfRange": true
```

Comment:
Reading the changesets docs:
> When set to true, Changesets will only bump peer dependents when peerDependencies are leaving the range.
Just to clarify if I understand this - does this just mean that it will leave our peers deps as is or it will bump the version within each major in that range

Reply:
From my understanding, it should leave peerDeps alone until a major outside the range is released, of which in that scenario we should be updating the peerDeps anyway.

I couldn't find an option to exclude only peerDeps from getting updates. It seemed to be an all-types-of-deps-or-nothing 😬

Reply:
Fair enough! lets merge it in and see 🤞😆

Reply:
For what it's worth, [Unified-Home](https://github.com/cultureamp/unified-home/blob/master/.changeset/config.json#L14) do this as well.